grammars/typescript.cson

```diff
@@ -711,12 +711,12 @@
         'name': 'meta.brace.curly.js'
     'patterns': [
       {
-        'include': '$self'
+        'include': '$base'
       }
     ]
 
   'type':
-    'begin': '(?=)'
+    'begin': '(?=[\\(\\{\\p{L}\\p{Nl}$_])'
```

Comment:
In some regexp implementations, that would work for internationalisation categories - matches letters or number letters.

Reference: http://xregexp.com/plugins/, no idea if Atom supports it personally, but I know they do fix some back references and other regexp features.